### ISSUE TITLE
missing artifact for darwin arm64

### ISSUE THREAD

**Issue by tannevaled** [hashicorp/consul-template#1564]
### Consul Template version

0.28.1

### Expected behavior

a pre-compiled artifact

### Actual behavior

not pre-compiled artifact

### Steps to reproduce

go to https://releases.hashicorp.com/consul-template/0.28.1/
there is an artifact for linux arm64 but not for darwin arm64.
consul have an artifact for darwin arm64.

**Comment by eikenb** [comment]
Hey @tannevaled,

Sorry for the missing binary. We are in the process of switching over to a new build system internally and I'll be using it for the 0.29.0 release which will support darwin-arm64. I am presently working on the [final bug fixes](https://github.com/hashicorp/consul-template/milestones/v0.29.0) for 0.29.0 and hope you  can understand my putting this off until that release so I can stay focused on those bugs and not distract the release engineers from handling issues with the new build system.

Thanks for the prompt though, I did double check about the darwin-arm64 support and I added this ticket to the release to make sure I don't forget about it.
